src/precice-aste-run.cpp

```diff
@@ -9,15 +10,13 @@ int main(int argc, char *argv[])
   auto context = aste::initializeMPI(argc, argv);
   auto options = getOptions(argc, argv);
 
-  if (options["verbose"].as<bool>()) {
-    boost::log::core::get()->set_filter(logging::trivial::severity >= logging::trivial::debug);
-  }
+  addLogSink(
+      options["verbose"].as<bool>() ? LogLevel::Verbose : LogLevel::Quiet,
+      options["all"].as<bool>() ? LogRankFilter::All : LogRankFilter::OnlyPrimary);
```

Comment:
Please add this option to the `docs/README`

Reply:
Done